stable/graylog/README.md

```diff
@@ -14,7 +14,7 @@ To install the Graylog Chart with all dependencies
 ```bash
 kubectl create namespace graylog
 
-helm install --namespace "graylog" -n "graylog" stable/graylog
+helm install --namespace "graylog" --name "graylog" stable/graylog
```

Comment:
Helm 2 requires `-n "graylog"` while Helm 3 uses without `-n`
I think Helm 3 is already standardize so it should be 

```
helm install --namespace "graylog" "graylog" stable/graylog
```